command-or-ctrl-key/src/index.ts

```diff
@@ -2,12 +2,22 @@
  * Get the appropriate keyboard shortcut key based on the user's platform.
  * Returns '⌘' for macOS and 'Ctrl' for other operating systems.
  */
-export const getCommandOrCtrlKey = () => {
-  const userAgent = navigator.userAgent.toLowerCase();
+export const getCommandOrCtrlKey = ({
```

Comment:
* create a `src/constants.ts` file
* Define a `DEFAULT_CONFIG = {.. your config}` 
* Import and pass it as a config `export const getCommandOrCtrlKey = (config = DEFAULT_CONFIG) =>{`
* update related tests as well